packages/javascript/bh-shared-ui/src/components/UpdateUserForm/UpdateUserForm.tsx

```diff
@@ -267,40 +260,42 @@ const UpdateUserFormInner: React.FC<{
                                         </Select>
                                     </FormControl>
                                 )}
                             />
                         </Grid>
 
-                        {authenticationMethod === 'saml' && (
+                        {authenticationMethod === 'sso' && (
                             <Grid item xs={12}>
                                 <Controller
-                                    name='SAMLProviderId'
+                                    name='SSOProviderId'
                                     control={control}
                                     rules={{
-                                        required: 'SAML Provider is required',
+                                        required: 'SSO Provider is required',
                                     }}
                                     render={({ field: { onChange, onBlur, value, ref } }) => (
                                         <FormControl>
-                                            <InputLabel id='SAMLProviderId-label' sx={{ ml: '-14px', mt: '8px' }}>
-                                                SAML Provider
+                                            <InputLabel id='SSOProviderId-label' sx={{ ml: '-14px', mt: '8px' }}>
+                                                SSO Provider
                                             </InputLabel>
                                             <Select
                                                 onChange={onChange as (event: SelectChangeEvent<string>) => void}
                                                 onBlur={onBlur}
-                                                value={value}
+                                                value={value?.toString()}
                                                 ref={ref}
-                                                labelId='SAMLProviderId-label'
-                                                id='SAMLProviderId'
-                                                name='SAMLProviderId'
+                                                labelId='SSOProviderId-label'
+                                                id='SSOProviderId'
+                                                name='SSOProviderId'
                                                 variant='standard'
                                                 fullWidth
-                                                data-testid='update-user-dialog_select-saml-provider'>
-                                                {SAMLProviders.map((SAMLProvider: any) => (
-                                                    <MenuItem value={SAMLProvider.id.toString()} key={SAMLProvider.id}>
-                                                        {SAMLProvider.name}
+                                                data-testid='update-user-dialog_select-sso-provider'>
+                                                {SSOProviders?.map((SSOProvider: any) => (
```

Comment:
Is TS still happy if you get rid of the `: any` here? I think the type should already be implicitly narrowed to just `SSOProviders[]`!

Reply:
Yup! Looks like `SSOProvider` works here, good catch!